web/src/api/software.ts

```diff
@@ -111,6 +111,12 @@ const probe = () => post("/api/software/probe");
 const register = ({ key, email }: { key: string; email?: string }) =>
   post("/api/software/registration", { key, email });
 
+/**
+ * Updates the URL for the registration
+ */
+const updateRegistrationUrl = (url: string) =>
+  put("/api/software/registration/url", url, { headers: { "Content-Type": "application/json" } });
```

Comment:
any reason why in this case the content-type is required where other parts do not need it? I think reason should be ideally part of the code so other readers who have same question will see it